lib/dao.js

```diff
@@ -430,15 +430,17 @@ function stillConnecting(dataSource, obj, args) {
  * otherwise, insert a new record.
  *
  * NOTE: No setters, validations, or hooks are applied when using upsert.
- * `updateOrCreate` is an alias
+ * `updateOrCreate` and `patchOrCreate` are aliases
  * @param {Object} data The model instance data
  * @param {Object} [options] Options for upsert
  * @param {Function} cb The callback function (optional).
  */
 // [FIXME] rfeng: This is a hack to set up 'upsert' first so that
 // 'upsert' will be used as the name for strong-remoting to keep it backward
 // compatible for angular SDK
-DataAccessObject.updateOrCreate = DataAccessObject.upsert = function upsert(data, options, cb) {
+DataAccessObject.updateOrCreate =
```

Comment:
Looks like you left a lot of empty spaces at the EOL, please remove them. I am surprised `eslint` didn't catch this.